aiida_common_workflows/cli/options.py

```diff
@@ -48,6 +49,21 @@ def get_spin_types():
     return [entry.value for entry in SpinType]
 
 
+class JsonParamType(click.ParamType):
+    """CLI parameter type that can load a JSON string into a python value."""
+    name = 'json'
+
+    def convert(self, value, param, ctx):
+        """Convert from a string to a python object."""
+        if not isinstance(value, str):
+            self.fail(f'{value!r} is not a valid string to be converted to json', param, ctx)
```

Comment:
```suggestion
        value = super().convert(value, param, ctx)
```
This will perform the validation for us